src/ofxKinectForWindows2/Source/Depth.h

```diff
@@ -20,6 +20,8 @@ namespace ofxKinectForWindows2 {
 				bool stitchFaces;
 				float facesMaxLength;
 				TextureCoordinates textureCoordinates;
+				float minDepth;

```

Comment:
I've got some more drastic changes coming for mesh handling. Basically changed the code flow to reuse a ofMesh object instead of clearing the memory and reassigning, makes the frame rate max out for me as opposed to a 1fps choke (confirmed that the issue is this with VS2012 profiler ). So the min/max depth are used in the mesh creation in front/back culling when constructing the mesh. I still have a few issues to sort out here though, mainly around the size of the mesh. 

At the moment it's easier to just use the camera space points but the units there are in metres, so maxes out at 4.5m. Returning this mesh means the image size is 4.5px deep which requires a bit of scaling which isn't ideal in my opinion. Currently toying around with the depth source access the color source to get the width and height, which I know are fixed but good practice to fetch them. Not really ideal having it cross classes but will see what I can do. Few solutions to find there.